packages/components/header/template.njk

```diff
@@ -1,167 +1,87 @@
-
-      <div class="nhsuk-header__content" id="content-header">
-        <div class="nhsuk-header__search">
-            <div class="nhsuk-header__search-wrap" id="wrap-search">
-              <form class="nhsuk-header__search-form" id="search" action="{{ params.searchAction if params.searchAction else 'https://www.nhs.uk/search/' }}" method="get" role="search">
-                <label class="nhsuk-u-visually-hidden" for="search-field">Search the NHS website</label>
-                <input class="nhsuk-search__input" id="search-field" name="{{ params.searchInputName if params.searchInputName else "search-field" }}" type="search" placeholder="Search" autocomplete="off" >
-                <button class="nhsuk-search__submit" type="submit">
-                  <svg class="nhsuk-icon nhsuk-icon__search" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
-                    <path d="M19.71 18.29l-4.11-4.1a7 7 0 1 0-1.41 1.41l4.1 4.11a1 1 0 0 0 1.42 0 1 1 0 0 0 0-1.42zM5 10a5 5 0 1 1 5 5 5 5 0 0 1-5-5z"></path>
-                  </svg>
-                  <span class="nhsuk-u-visually-hidden">Search</span>
-                </button>
-              </form>
-            </div>
-          </div>
-      </div>
 
-    </div>{# close nhsuk-header__container #}
-    <div class="nhsuk-navigation-container">
-      <nav class="nhsuk-navigation" id="header-navigation" role="navigation" aria-label="Primary navigation">
-        <ul class="nhsuk-header__navigation-list {%- if params.primaryLinks.length < 4 %} nhsuk-header__navigation-list--left-aligned{% endif %}">
-          {%- for item in params.primaryLinks %}
-          <li class="nhsuk-header__navigation-item {%- if item.classes %} {{ item.classes }}{% endif %}" {%- for attribute, value in item.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
-            <a class="nhsuk-header__navigation-link"  href="{{item.url}}">
-              {{item.label}}
-            </a>
-          </li>
-          {%- endfor %}
-          <li class="nhsuk-header__navigation-item nhsuk-header__navigation-item--home">
-            <a class="nhsuk-header__navigation-link"  href="/">
-              Home
-            </a>
-          </li>
-          <li class="nhsuk-mobile-menu-container">
-            <button class="nhsuk-header__menu-toggle nhsuk-header__navigation-link" id="toggle-menu" aria-expanded="false">
-              <span class="nhsuk-u-visually-hidden">Browse</span>
-              More
-              <svg class="nhsuk-icon nhsuk-icon__chevron-down" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
-                <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
-              </svg>
-            </button>
-          </li>
-        </ul>
-      </nav>
+{%- if params.transactionalService and not params.primaryLinks and not params.search %}
```

Comment:
```suggestion
{%- if params.transactionalService %}
```

I think this should just check for the presence of `transactionalService`? This would fix an existing bug which is that currently if you try and have a transactional service name AND navigation items, only the navigation items appear but not the service name.

Reply:
I spotted this – there’s a lot of logic being used that might be best dealt with in styles or a broader review of how the different parts of the header interact with each other – I didn’t want to change any of that behaviour.

But if we think this is the right place to make this change, happy to do so.

Reply:
given we're changing the logic anyway (to test on presence of navigation items and search rather than the booleans) I think we can fix it here? Feels like an (existing) bug.

Reply:
Okay, I have includes this change in a separate commit, currently 94b9ee14962cdd6029ca41fd095a28f5d4d1ad36